Pulseway/readme.md

```diff
@@ -0,0 +1,13 @@
+Huntress Deployment PowerShell v2 scripted wrapped within a function for deploying Huntress with Pulseway RMM.
+
+Usage:
+
+Install-Huntress [-acctkey <account_key>] [-orgkey <organization_key>] [-tags <optional_tags_here>] [-reregister] [-reinstall] [-uninstall] [-repair]
+
+Examples:
+
+ -acctkey 1234abc -orgkey hackerhunter
+
+ -acctkey 1234abc -orgkey hackerhunter -tags "" -reinstall $false -reregister $false -uninstall $false -repair $false
+
+]
```

Comment:
@cameron-huntress I think something is odd with the formatting here of the .md . Did you mean for this to be in a code block?

![image](https://user-images.githubusercontent.com/50087150/236338283-53f30ab8-e698-469b-a333-9d8ab3db5b75.png)